### ISSUE TITLE
Text between apostrophes are incorrectly colored

### ISSUE THREAD

**Issue by rsyring** [gigapixel/atom-language-gherkin#9]
```
Scanario: invalid order identifier
    Given an invalid order identifier in the URL
    When I visit the order details page
    Then I'll receive a friendly message that informs the order isn't valid
```
When syntax highlighting the last line, the middle part of the line is green from the first apostrophe to the second.

**Comment by rich13** [comment]
I second this :+1:

**Comment by gigapixel** [comment]
Fixed on v1.0.3 please check.

**Comment by rich13** [comment]
Fixed for me - thanks :-)
